src/main/resources/messages/messages.properties

```diff
@@ -326,3 +326,14 @@ WISH.DETAILS.MARKET.FLOW.PARAGRAPH=Lorem ipsum dolor sit amet, consectetur adipi
 CONTACT.US.TITTLE= Contact Us
 CONTACT.US.SEND.BUTTON=Send
 
+# Messages
+MESSAGES.PAGE.TITLE= Messages
+MESSAGES.NAV.HOME= Home

```

Comment:
mesajul deja exista la cheia NAV.HOME, dar e duplicat in mai multe locuri. Grupeaza toate mesajele legate de navigarea in pagina in acelasi loc (sectiunea #navigation). Foloseste NAVIGATION in loc de NAV. Sterge spatiul dintre = si mesajul efectiv.